atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/CommitLockTest.java

```diff
@@ -215,4 +166,23 @@ private LockResponse lock(LockDescriptor lockDescriptor) {
         LockRequest lockRequest = LockRequest.of(ImmutableSet.of(lockDescriptor), 5_000);
         return timelockService.lock(lockRequest);
     }
+
+    private TransactionManager createTransactionManager(ConflictHandler conflictHandler) {
+        ImmutableMap<TableReference, ConflictHandler> tablesToWriteWrite = ImmutableMap.of(
```

Comment:
nit: We probably shouldn't name this map this way, as the tables here aren't actually being mapped to the write-write strategy generally speaking. Maybe `tablesToConflictHandlers`? That said, we might not even need this map as a thing (see next comment).